components/nimbus/Cargo.toml

```diff
@@ -39,6 +39,7 @@ unicode-segmentation = "1.8.0"
 error-support = { path = "../support/error" }
 remote_settings = { path = "../remote_settings", optional = true }
 cfg-if = "1.0.0"
+glean = "54.0.0"
```

Comment:
So we use Glean officially now — I tried using the path to the git submodule but Glean throws a bunch of warnings that would end up breaking the Rust clippy tests. So I went with manually defining it here.

Alternatively do we need to do the path ref and either fix/ignore the warnings?

Reply:
jk lol apparently it broke the dependency tree anyway

Reply:
I use it in tests now 😬